### ISSUE TITLE
getLongName

### ISSUE THREAD

**Issue by rfoster93** [refundable-tgm/huginn#7]
@mbeier-tgm please implemented /getLongName?name='cbrein'

Input:
```
short name of teacher
```

Output:
example
```
{
    short: 'cbrein',
    long: 'Christoph Brein'
}
```

**Comment by mbeierat** [comment]
implemented basis for a further implementation in 8984b5f

**Comment by mbeierat** [comment]
final implementation introduced in c92b882

The endpoint `/getLongName` responds to GET-Requests when a query parameter `name` containing the short name of the teacher and a valid auth token is provided.

The response of the endpoint looks as follows: 

```
{
    "short": "cbrein",
    "long": "Christoph Brein",
    "uuid": "5fa7d0ed-a02e-4e09-8eec-a374d301874b"
}
```
